.github/workflows/check.yaml

```diff
@@ -0,0 +1,20 @@
+on: [push, pull_request]
+name: Check
+jobs:
+  check:
+    strategy:
+      matrix:
+        go-version: [1.15.x]
```

Comment:
Should we simply bump it to `1.16.x` while we are at it?

Reply:
It's not a simple bump; compilation starts failing